### ISSUE TITLE
TCPDF doesn't load http://www.mydomain.com image if domain is https

### ISSUE THREAD

**Issue by itajackass** [tecnickcom/TCPDF#155]
I've an tinymce editor on my PHP webapp to save "precompiled" content (text/images).

I've saved an old entry with some text and a image link. This is the HTML view of tinymce editor.

    <p>This is an image</p>
    <img src="http://mywebapp.com/files/image_1.jpg" />

Then a button named "Create PDF and SEND"...create the A4 pdf version of my tinymce content (using TCPDF) and send via email.

All ok when my app was on http. Now we migrated to http**s** and when i generate PDF, image disappear from PDF....i need to manually change (in html view of tinymce editor textarea):

    <img src="http://mywebapp.com/files/image_1.jpg" />

to:

    <img src="https://mywebapp.com/files/image_1.jpg" />

adding **s** to src. there is a solution to fix this problem on TCPDF?

**Comment by NeoTechni** [comment]
No, this is by design of http/https itself. Not an issue with TCPDF.

You should use either <img src="/files/image_1.jpg" />
Or make a function to get the proper webroot, ie:
<img src="<?= webroot(); ?>files/image_1.jpg" />

**Comment by HariharanUmapathi** [comment]
@itajackass does the explanation from @NeoTechni  resolved your issue ?

**Comment by nicolaasuni** [comment]
Thank you for reporting this. TCPDF 6 is now a legacy project and is officially **deprecated**, so no further fixes or features will be merged into this repository. As part of this transition, we are closing older issues. Please migrate to **[tc-lib-pdf](https://github.com/tecnickcom/tc-lib-pdf)** and, if the same problem still occurs there, open a new issue with updated reproduction steps. For more context, see the pinned announcement: [Thank You - and the Future of TCPDF](https://github.com/tecnickcom/TCPDF/issues/867).
